sway/main.c

```diff
@@ -413,6 +413,8 @@ int main(int argc, char **argv) {
 		goto shutdown;
 	}
 
+	gain_realtime();
```

Comment:
Bikeshed: Something like `set_proc_scheduling` or something else to be clearer that we're fixing process scheduling, whatever that requires.

Reply:
Sounds good.

Reply:
I renamed this to `set_rr_scheduling`.